src/metricscollector/config/config.go

```diff
@@ -54,12 +55,21 @@ var defaultCollectorConfig = CollectorConfig{
 	PollInterval:    DefaultPollInterval,
 }
 
+type DataPruneConfig struct {
+	CutoffDays int `yaml:"cutoff_days"`
+}
+
+var defaultDataPruneConig = DataPruneConfig{
```

Comment:
Just so you know, I am not convinced I want the pruning to be done by the metricscollector.

I think we will have a standalone pruner that will do it for all the DBs. So this code should just move to src/autoscaler/pruner as the top level package. The pruners themselves can live at the root and the main would be autoscaler/pruner/cmd/pruner.